autodispose-android-archcomponents/src/main/java/com/uber/autodispose/android/lifecycle/AndroidLifecycleScopeProvider.java

```diff
@@ -73,8 +73,8 @@ public static AndroidLifecycleScopeProvider from(LifecycleOwner owner) {
    * @param untilEvent the event until the scope is valid.
    * @return a {@link AndroidLifecycleScopeProvider} against this owner.
    */
-  public static AndroidLifecycleScopeProvider from(LifecycleOwner owner,
-                                                   Lifecycle.Event untilEvent) {
+  public static AndroidLifecycleScopeProvider from(
```

Comment:
One more nit - if they start on a new line, let's make each of them a separate line

Reply:
Like this?

```java
public static AndroidLifecycleScopeProvider from(
          LifecycleOwner owner, 
          Lifecycle.Event untilEvent) {
```

Reply:
yep!

Reply:
Done!